test/instrumentation/BUILD

```diff
@@ -44,7 +45,7 @@ genrule(
         "//:all-srcs",
     ],
     outs = ["stable-metrics-list.yaml"],
-    cmd = "./$(locations :instrumentation) $(locations //:all-srcs) > $@",
+    cmd = "echo '$(locations //:all-srcs)' | ./$(locations :instrumentation) - > $@",
```

Comment:
could we echo one to a line from the shell side instead?

Reply:
Do you mean like?
`./$(locations :instrumentation) - <(echo '$(locations //:all-srcs)') > $@`

Reply:
Sorry, I don't understand your intention. Can you give example?

Reply:
Er, something like `for loc in $(locations //:all-srcs); do echo "$loc"; done | ./$(locations :instrumentation) -` ... then we don't need to lex on the go side... one input per line

Reply:
Thanks for suggestion! it's much better